bom/pom.xml

```diff
@@ -1004,6 +1004,16 @@
                 <artifactId>helidon-integrations-oci-vault-health</artifactId>
                 <version>${helidon.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.helidon.integrations.oci</groupId>
+                <artifactId>helidon-integrations-oci-metrics</artifactId>
+                <version>${helidon.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.helidon.integrations.oci</groupId>
+                <artifactId>helidon-integrations-oci-metrics-cdi</artifactId>
```

Comment:
Note that my OCI CDI integration work is in `integrations/cdi`.

Reply:
Yep, I'm aware of `integrations/cdi/oci` for the universal OCI integration. This one is specific to the OCI Metric bridge CDI processing in https://github.com/oracle/helidon/pull/4003/files#diff-36b8cc7f1e880a4432c555a4fa92d499cff186e471e0cd6105e89e9a09b5fa03 which includes getting an instance of the MonitoringClient and triggering the executorService for this.

Reply:
Right; I guess my point is: where do CDI integrations of various kinds go?  Do they go under `integrations/some-tech-here/cdi` or do they go under `integrations/cdi/some-tech-here`?

Reply:
Many other components that have a non-CDI and a CDI piece use the pattern

`/helidon/_component_/_component_` - non-CDI part
`/helidon/_component_/cdi` - CDI part

E.g., health, metrics, Micrometer integration, vault integration, etc.

We should probably decide as a group which way we want to go and then reconcile any outliers.

Reply:
Indeed `integrations/vault` uses the same approach we did: 
```
~/helidon_project/helidon-2.x/integrations$ ls -l vault
total 16
-rw-r--r--   1 klustria  staff   1.8K Mar 24 00:56 README.md
drwxr-xr-x   8 klustria  staff   256B Mar 24 01:35 auths/
drwxr-xr-x   5 klustria  staff   160B Mar 24 01:35 cdi/
-rw-r--r--   1 klustria  staff   1.5K Mar 24 01:02 pom.xml
drwxr-xr-x  10 klustria  staff   320B Mar 24 01:35 secrets/
drwxr-xr-x   5 klustria  staff   160B Mar 24 01:35 sys/
drwxr-xr-x   3 klustria  staff    96B Mar 24 01:35 target/
drwxr-xr-x   5 klustria  staff   160B Mar 24 01:35 vault/
```
@ljnelson , looks like `integrations/some-tech-here/cdi` is already a pattern being used so if it's ok with you, we'll stick with that same pattern.

Reply:
+1 on this from my perspective fwiw ;-)